cdc/model/sink.go

```diff
@@ -478,38 +478,83 @@ type RedoDDLEvent struct {
 
 // FromJob fills the values of DDLEvent from DDL job
 func (d *DDLEvent) FromJob(job *model.Job, preTableInfo *TableInfo) {
+	if d.Type == model.ActionRenameTables {
```

Comment:
Could you add some comments why it skips `RenameTables`?

Reply:
> Could you add some comments why it skips `RenameTables`?

Done.